api/python/provisioner/commands/upgrade/set_swupgrade_repo.py

```diff
@@ -832,6 +841,14 @@ def dynamic_validation(self, params, targets: str,
 
             repo.release = release
 
+            if (not dry_run and version.parse(release)
```

Comment:
please activate that for dry run as well

Reply:
Done